.github/workflows/kb_sdk_test.yaml

```diff
@@ -59,4 +59,9 @@ jobs:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
       run: |
         sh $GITHUB_WORKSPACE/kb_sdk_actions/bin/kb-sdk test
-        bash <(curl -s https://codecov.io/bash)
+
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```

Comment:
we could use `v4` but that's ok.

Reply:
Both myself and Bill had failures with v4 so I've been holding off for now.